### ISSUE TITLE
Support quantiles with Summary Metric for Amazon CloudWatch interoperability

### ISSUE THREAD

**Issue by alolita** [open-telemetry/opentelemetry-specification#1145]
**What are you trying to achieve?**

We have a use case for reporting min, max and arbitrary quantile values (e.g. p99) recorded over an interval in the OTLP [metrics.proto](https://github.com/open-telemetry/opentelemetry-proto/blob/master/opentelemetry/proto/metrics/v1/metrics.proto). Our use case was fully supported earlier by the Summary datatype which was removed in https://github.com/open-telemetry/opentelemetry-proto/pull/199.

Currently there is no clean way of even reporting the min and max values, however that may become possible with issue [982](https://github.com/open-telemetry/opentelemetry-specification/issues/982). We additionally require support for reporting arbitrary quantiles.

**What did you expect to see?**

We would like a form of the Summary data type to be reintroduced into OTLP capable of supporting the reporting of arbitrary quantiles. When Summary was removed the change was described as temporary. What is the current timeline for its reintroduction?

**Additional context.**

As mentioned above we believe that a viable alternative for min/max may be to have them transported in the Histogram datatypes as one of the options in issue [982](https://github.com/open-telemetry/opentelemetry-specification/issues/982).

The other way we have considered it to encode them as exemplars in the histogram format. While possible for min and max
 
```
{
 0, // this is either the min or max ( min in this case)
 timestamp
},
{
 100, // this is either the min or max ( max in this case)
 timestamp
}
```
 
it seems less reasonable for arbitrary quantiles as we have no easy way of encoding which percentile the value corresponds to.
 
 cc: @bogdandrutu @jmacd @tigrannajaryan @amanbrar1999 @JasonXZLiu

**Comment by gcacace** [comment]
Proposal in open-telemetry/opentelemetry-proto#227

**Comment by gcacace** [comment]
Please consider the main use case for `IntSummary` and `DoubleSummary` is for transport purposes (not aggregation). In this specific case I've removed `AggregationTemporality`, addressing concerns expressed in open-telemetry/opentelemetry-proto#199.

I've added `min` and `max`.

Not sure if we need to support both `IntSummary` and `DoubleSummary`, or the Double version is enough (I've included both for model completeness).

Not sure if we need to support `exemplars`.

**Comment by weyert** [comment]
Can't this ticket be closed as open-telemetry/opentelemetry-proto#227 got merged?
